source/source-connector/configuration-properties/change-stream.txt

```diff
@@ -82,10 +82,19 @@ Settings
      - | **Type:** boolean
        |
        | **Description:**
-       | Whether to publish the changed document instead of the full change
-         stream document. When set to ``true``, the connector
-         automatically sets the ``change.stream.full.document`` property to
-         ``updateLookup`` to receive the updated documents.
+       | Whether to return only the ``fullDocument`` field from the full
+         change stream document. The ``fullDocument`` field contains the
+         most current version of the updated document. When set to
+         ``true``, the connector automatically sets the
+         ``change.stream.full.document`` property to ``updateLookup`` to
+         receive the updated documents.
```

Comment:
Suggestion:

The transition between the definition of fullDocument back to the setting value seems abrupt to me. I think starting a new paragraph between these sentences would be helpful.